gdal/frmts/idrisi/IdrisiDataset.cpp

```diff
@@ -2815,6 +2815,10 @@ CPLErr IdrisiGeoReference2Wkt( const char* pszFilename,
     {
         oSRS.SetSinusoidal( dfCenterLong, dfFalseEasting, dfFalseNorthing );
     }
+    else if( EQUAL( pszProjName, "CylindricalEA")) || EQUAL( pszProjName, "Cylindrical Equal Area" ))
```

Comment:
Hum, did you actually test your pull request ? This fails to build because of:
```suggestion
    else if( EQUAL( pszProjName, "CylindricalEA") || EQUAL( pszProjName, "Cylindrical Equal Area" ))
```

Reply:
Will modify and test again. Got to go now because I have another test - an appointment for covid-19 test.

Reply:
I've pushed the build fix